tasks/Functions-client/request.js

```diff
@@ -25,7 +25,13 @@ task("functions-request", "Initiates a request from an Functions client contract
         maxFeePerGas: ethers.utils.parseUnits("50", "gwei"),
         gasLimit: 500000,
       }
-    }  
+    }
+
+    if (network.name === "mumbai") {
+      overrides = {
+        gasLimit: 500000,
```

Comment:
You could just set it globally for all networks.

Reply:
Just added this w/ a comment